ResearchKit/ActiveTasks/ORKActiveStepQuantityView.m

```diff
@@ -245,12 +242,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
         [self addSubview:_leftView];
         [self addSubview:_rightView];
         [self addSubview:_metricKeyline];
-        [self setupConstraints];
+        [self setUpConstraints];
```

Comment:
This `setUpConstraints` call is on the `init` method, and `updateConstraints` doesn't call `setUpConstraints`. I don't think there's an infinite loop, could you re-check?